build.gradle

```diff
@@ -60,7 +60,7 @@ dependencies {
     // for the ASCII Trees
     testImplementation 'hu.webarticum:tree-printer:+'
 
-    testImplementation 'org.duckdb:duckdb_jdbc:1.0.0'
+    testImplementation 'org.duckdb:duckdb_jdbc:+'
```

Comment:
This is correct in general, however DUCKDB 1.1.1 introduced a CSV related bug which affects our test suite.
DuckDB fixed that already but we would need to wait for the next release.